Sources/SnapshotAssertion/Cocoa.swift

```diff
@@ -0,0 +1,44 @@
+import Cocoa
```

Comment:
Snuck some preliminary support for Cocoa stuff here. Means we can maybe render some `NSWebView` stuff more quickly for `Html`/`Css`.